### DISCUSSION TITLE
LLama3 on koboldcpp?

### DISCUSSION THREAD

**Discussion by Pablo4096** [YellowRoseCx/koboldcpp-rocm#52] [General]
Is it possible to run LLama3 with koboldcpp? When trying to run Meta-Llama-3-8B-Instruct-bf16, the program reports an error. 
RX6600, 32GB RAM 
![kobold](https://github.com/YellowRoseCx/koboldcpp-rocm/assets/167965964/b22a7185-ea2c-4830-b1ca-d62cbf430283)

**Comment by YellowRoseCx** [comment]
It may have to do with that version being quantized to bf16, try one of these https://huggingface.co/QuantFactory/Meta-Llama-3-8B-Instruct-GGUF/tree/main
